sinks/gcmautoscaling/driver.go

```diff
@@ -52,11 +52,19 @@ type utilizationMetric struct {
 var autoscalingMetrics = map[string]utilizationMetric{
 	"cpu/usage": {
 		name:        "cpu/node_utilization",
-		description: "Cpu utilization as a percentage of node capacity",
+		description: "Cpu utilization as a share of node capacity",
```

Comment:
I mean total usage divided by node capacity. It's the number from range [0, 1] without any unit.